build.go

```diff
@@ -277,7 +277,7 @@ func build() {
 	if runtime.GOOS == "windows" {
 		artifact = "octant.exe"
 	}
-	runCmd("go", nil, "build", "-o", "build/"+artifact, GO_FLAGS, "-v", "./cmd/octant")
+	runCmd("go", nil, "build", "-mod=vendor", "-o", "build/"+artifact, GO_FLAGS, "-v", "./cmd/octant")
```

Comment:
shouldn't be necessary with go 1.14+

Reply:
I thought the same, but was wondering why Go was no pulling the proper set of dependencies.

Reply:
the go.mod file was set to 1.13, and is now 1.15 on master